server/src/main/java/org/tctalent/server/util/NextStepHelper.java

```diff
@@ -30,12 +30,16 @@ public class NextStepHelper {
     /**
      * Format used for date timestamp in next step audit stamp
      */
-    public static DateTimeFormatter nextStepDateFormatter = DateTimeFormatter.ofPattern("yyMMdd");
+    public static DateTimeFormatter nextStepDateFormatter = DateTimeFormatter.ofPattern("ddMMMuu");
 
     /**
-     * The audit stamp starts with this string
+     * The audit name stamp starts with this string
      */
-    public static String nextStepAuditStampDelimiter = " --";
+    public static String nextStepAuditNameDelimiter = " --";
+    /**
+     * The audit date ends with this string
+     */
+    public static String nextStepAuditDateDelimiter = "| ";
 
     /**
      * If the requested next step is different from the current next step, the next step
```

Comment:
This JavaDoc needs to be updated. It talks about the audit stamp being added to the end - update to clarify what the new audit stamp is - ie date at front and user and end.

Reply:
Updated